physionet-django/console/static/console/js/search-console.js

```diff
@@ -0,0 +1,16 @@
+// Script to search lists in the console
+// e.g. search users, news items, etc.
+
+function search(url, value){
+ var csrftoken = getCookie('csrftoken');
+ $.ajax({
+         type: "POST",
+         url: url,
+         data: {'csrfmiddlewaretoken':csrftoken,
+                'search': value
+         },
+         success: function reloadSection(result){
+             $("#searchitems").html(result);
+         },
+ });
+}
```

Comment:
data = {'csrfmiddlewaretoken':csrftoken,  'search': value}
$( "#searchitems" ).load(url, data);

Reply:
Thanks, Lucas - added  in https://github.com/MIT-LCP/physionet-build/pull/799/commits/feb610040a6faa3bedec5b2c05ccb78d911b4be2